packages/styles/src/makeTheme.ts

```diff
@@ -18,3 +19,42 @@ export const makeTheme = <Mode extends string = string>(
 
   return newTheme;
 };
+
+/** Takes in a Record of `HvThemeColors`, where the values can */
```

Comment:
_"where the values can"_ is a great cliffhanger 😄

Reply:
oops, updating docs is hard (or boring) 😅